src/Tests/Microsoft.Diagnostics.Monitoring.Tool.UnitTests/TriggerTestsHelper.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Diagnostics.Tools.Monitor.CollectionRules.Options;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Options;
+
+namespace Microsoft.Diagnostics.Monitoring.Tool.UnitTests
+{
+    internal static class TriggerTestsHelper
```

Comment:
Consider unifying LimitsTestsHelper and TriggerTestsHelper into a more generic helper that gets Options.

Reply:
Presumably then we'd also pull `GetActionOptions` out of ActionTestsHelper.cs - I could put all three of these helpers together in a new class, but if we want to add more helper methods for limits/triggers in the future, then we'll end up with everything categorized by actions/limits/triggers, with the exception of getting options, which would be in its own bubble. I don't have an issue either way - just more of a question of how we want to keep test helpers organized.